core/msg_bus.c

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright (C) 2020 ML!PA Consulting GmbH
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     core_msg
+ *
+ * @{
+ *
+ * @file
+ * @brief       Messaging Bus API for inter process message broadcast.
+ *
+ * @author      Benjamin Valentin <benjamin.valentin@ml-pa.com>
+ *
+ * @}
+ */
+
+#include "irq.h"
+#include "msg_bus.h"
+
+void msg_bus_init(msb_bus_t *bus)
```

Comment:
maybe make this return the msgbus count?

Reply:
Not sure what the user would do with that though.

Reply:
ok